app/grants/clr.py

```diff
@@ -293,6 +293,9 @@ def predict_clr(random_data=False, save_to_db=False):
         if save_to_db:
             grant.clr_prediction_curve = list(zip(potential_donations, potential_clr))
             grant.save()
+            base = grant.clr_prediction_curve[0][1]
```

Comment:
@danlipert this adds a third array element, which is how much a specific contribution will net affect the clr match (as opposed to the previous index which is the total if they did)

Reply:
<img width="294" alt="Screenshot 2019-09-23 at 8 25 28 PM" src="https://user-images.githubusercontent.com/5358146/65437477-7d6c4980-de41-11e9-88d9-94cda7792436.png">

@owocki Ah this would mean we'd also have to update fund.js  cause `grant.clr_prediction_curve` 
is gonna change from `[[amount, clr_match]]` -> `[amount, clr_match_increased_from_current]`

Reply:
i dont think so. the new schema is 

```
[[amount, clr_match, clr_match_increased_from_current]
```
not
```
[amount, clr_match_increased_from_current]
```

Reply:
Ohh :P lol okie makes sense